cmd/flags_core.go

```diff
@@ -50,6 +51,10 @@ func ParseCoreFlags(cmd *cobra.Command, env *Env) error {
 
 	grpcEndpoint := cmd.Flag(coreGRPCFlag).Value.String()
 	if grpcEndpoint != "" {
+		// trim ip:port from the endpoint
+		grpcEndpoint = strings.Trim(grpcEndpoint, "http://")  //nolint:staticcheck
+		grpcEndpoint = strings.Trim(grpcEndpoint, "https://") //nolint:staticcheck
+		grpcEndpoint = strings.Trim(grpcEndpoint, "tcp://")   //nolint:staticcheck
```

Comment:
This seems rather...hacky. Isn't there some URL library that can be used to trim these properly?

Reply:
There is. https://pkg.go.dev/net/url. @renaynay